src/database/index.js

```diff
@@ -34,23 +37,23 @@ const createKnex = async ({ sqlite, postgres }) => {
           password: postgres.password,
           database: postgres.database,
           ssl: postgres.ssl
-        },
-        useNullAsDefault: true
-      })
+        }
+      }
```

Comment:
@emirotin , to avoid `if/else`-s and make code shorter you could do something like:
```
      config = {
        client: 'pg',
        connection: postgres.connection || _.pick(postgres, ['host', 'port', 'user', 'password', 'database', 'ssl'])
      }
```

Reply:
Yeah I've considered that, then left the code mostly unchanged as this change is outside of the scope of the PR, so I've only replaced dynamic require with static import as `knex` is needed in every branch. But still nice to have so will do in a separate commit later.